python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -672,8 +688,9 @@ def addRun(self, run):
         # TODO: this check is probably undesirable, as we may want to have
         # multiple Runs output to the same collection.  Fixing this requires
         # (at least) modifying getRun() accordingly.
-        selection = select([exists().where(runTable.c.collection == run.collection)])
-        if self._connection.execute(selection).scalar():
+        selection = select([func.count(1)]).select_from(runTable).where(runTable.c.collection ==
```

Comment:
Just `count()` (which is `COUNT(*)`) is probably more common pattern for row counting.